restful-api.md

```diff
@@ -1192,6 +1199,14 @@ Check trading credit balance.
 }
 ```
 
+# Additional
+For the  use of `cur`(currency) for any APIs request. Please be cautious of these cryptocurrency when you specified on the request.
+
+Name|Currency
+---|---
+Terra Classic|`LUNA`
+Terra 2.0|`LINA2`
```

Comment:
`LINA` I think is a hero in dota. Please revisit this again.

Reply:
my bad